WcaOnRails/app/models/regional_organization.rb

```diff
@@ -5,8 +5,9 @@ class RegionalOrganization < ApplicationRecord
   has_one_attached :bylaws
   has_one_attached :extra_file
 
-  scope :currently_acknowledged, -> { where("end_date IS NULL OR end_date > ?", Date.today) }
-  scope :not_currently_acknowledged, -> { where("end_date < ?", Date.today) }
+  scope :currently_acknowledged, -> { where("start_date IS NOT NULL AND end_date IS NULL OR end_date > ?", Date.today) }
+  scope :pending_approval, -> { where("start_date IS NULL") }
+  scope :previously_acknowledged, -> { where("start_date IS NOT NULL AND end_date < ?", Date.today) }
```

Comment:
I wonder how `NULL` `end_date` is handled here. Even if it works due to some conversions, I believe it would make sense to be explicit here and add `end_date IS NOT NULL`.

Reply:
Ok, I'm adding it.